### PR TITLE
Add Kubescape proposals for Term 2

### PR DESCRIPTION
**Pull request by craigbox**
* [Store Kubescape configuration scan results as CRs](https://github.com/kubescape/kubescape/issues/1225)
* [Prometheus exporter for image vulnerabilities](https://github.com/kubescape/kubescape/issues/1226)
* [Vulnerability-based Dockerfile generator](https://github.com/kubescape/kubescape/issues/1227)

### CODE REVIEW

File: programs/lfx-mentorship/2023/02-Jun-Aug/project_ideas.md [hunk 1]
```diff
@@ -101,6 +102,40 @@ Test coverage could increase
 - Mentor(s): Reto Lehmann @ReToCode (rlehmann AT redhat DOT com),  Stavros Kontopoulos @skonto (skontopo AT redhat DOT com)
 - Upstream Issue (URL): https://github.com/knative/serving/issues/12718
 
+### Kubescape
+
+#### Store Kubescape configuration scan results as CRs
+
+- Description: Implement a mechanism in the Kubescape operator which detects changes in the cluster and triggers a scan of the cluster.
```
Comment (aliok) [inline programs/lfx-mentorship/2023/02-Jun-Aug/project_ideas.md:109]: Could you add a bit of more description here @craigbox ? Why this is beneficial? What are the current problems?

**Review by aliok — COMMENTED** [review]
Thanks for volunteering to mentor contributors @slashben @craigbox @dwertent !

I think applicants would appreciate more description and I added 2 comments on the PR about that.

File: programs/lfx-mentorship/2023/02-Jun-Aug/project_ideas.md [hunk 2]
```diff
@@ -101,6 +102,40 @@ Test coverage could increase
...
+- Mentors:
+  - Ben Hirschberg (@slashben, ben AT armosec.io)
+  - Craig Box (@craigbox, craigb AT armosec.io)
+  - David Wertenteil (@dwertent, dwertent AT armosec.io)
+ - Upstream Issue: https://github.com/kubescape/kubescape/issues/1225
+
+#### Prometheus exporter for image vulnerabilities
+
+- Description: Develop a Prometheus exporter for image vulnerability information produced by Kubescape.
+- Expected Outcome: The ability to alert on vulnerabilities in a cluster based on Prometheus data
```
[the remark this answers is not in the document]
Comment (craigbox) [reply programs/lfx-mentorship/2023/02-Jun-Aug/project_ideas.md:121]: Done

File: programs/lfx-mentorship/2023/02-Jun-Aug/project_ideas.md (hunk 1 above)
Comment (craigbox) [reply programs/lfx-mentorship/2023/02-Jun-Aug/project_ideas.md:109]: Done

**Review by nate-double-u — APPROVED** [review]
Thanks for the updates

**Pull request merged**